util/i2csvg/README.md

```diff
@@ -0,0 +1,159 @@
+# i2csvg -- Generate svg pictures of i2c commands from text description
+
+i2csvg is a Python tool to read i2c transactions from a text file and
+generate `svg` pictures of them.
+
+Two source representations are supported.
+The first is a more readable format and describes transactions in text.
+The second is a hex-dump of values written to the I2C fifo.
+In either format a line starting with a `#` is taken as a comment and a line staring with a `T` will be used as a title in the output.
```

Comment:
We're attempting to standardize on 80 char READMEs, one sentence per line on other `.md` files.

Reply:
wrapped